docs/research/wallet-comparison.md

```diff
@@ -0,0 +1,171 @@
+  - Run own LND node on Phone ([Eclair](https://github.com/ACINQ/eclair), ACINQ in-house lightning implementation)
+  - Channel management through ACINQ nodes
+  - Can configure own Electrum Server
+  - Cannot configure own Lightning node (outside the phone setup)
+- Where is trust involved?
+  - Initial channel opening
+  - Onboarding with BTC
+
+Pro:
+
+- Good looking and efficient UI and UX
+- One of the best onboarding UX out there
+  - One-click wallet setup
+  - Onboarding using BTC
+- BIP39 (+BIP84) 12-word wallet backup
+
+Pro/Con:
+
+- Channels management by ACINQ
+  - Slightly higher costs for more convenience
+- Channel backup managed by ACINQ
+  - "Minimal" trust for more convenience
+
+Con:
+
+- Hard to understand the underlying constraints for the Lightning setup
+- Costs for transactions are not transparent (retrospectively)
+  - Example
+    - Initially funded the wallet via BTC, had BTC worth `50.3` USD incoming.
+    - Once they were actually confirmed it changed to `49.95` USD because the fees were deducted.
+    - These fees are never visible again, when going to the details I cannot see how much fee was actually deducted and what for.
+    - A better cost breakdown would be nice to have.
+
+For details check these links:
+
+- https://medium.com/@ACINQ/introducing-phoenix-5c5cc76c7f9e
+- https://phoenix.acinq.co/faq
+
+### Blue Wallet
+
+In a nutshell: Semi- or Non-custodial (when using own LND) setup and a good UX that includes multiple wallet, but overall not as good as Phoenix.
+
+Setup:
+
+- Plug in own LND node, or use [LndHub](https://github.com/BlueWallet/LndHub)
+  - LndHub is a semi-custodial setup ("minimum trust for endusers")
+- Where is trust involved?
+  - LndHub
+
+Pro:
+
+- Multiple wallet support
+- Vaults (Not tried out, feels like a different feature unrelated to Lightning Wallet)
+
+Pro / Con:
+
+- One-click wallet create for setup, but onboaring not as clear and rich as Phoenix
+
+Con:
+
+- Cannot "empty the wallet", always have to keep at least some sats in for "paying fees"
+  - This is super annoying, it is literally impossible to empty this wallet.
+- Multi-wallet management
+  - The wallets are completely independent; sending money from one to the other is quite cumbersome.
+  - I would have expected that I can easily move funds from one wallet to another, but this is quite cumbersome.
+  - Not sure what the use cases behind multiple wallets is; it does not feel like you average "bank account" app where you divide your money into different accounts for different purpose.
+- Send / Receive are not as intuitive as for Phoenix
+  - When using Phonex it becomes clear how Lightning works, even when one is new to send/receive and creating/paying invoices.
+  - The Blue wallet interface can be confusing, because it is not clear what one has to do to send/receive.
+- When using LndHub the backup is an lndhub link or a QR code. There is not 12-word seed available that can just be copied.
```

Comment:
not sure whether this is a con, it might a security feature. clipboards (especially on Android) are not too safe to use, because other apps might reach into that without your knowledge ;/ (on iOS you get notified if an app wants to use clipboard).

Reply:
I don't want to use the phone clipboard - I want to be able to write it on a piece of paper without using any other tech ;)